std_msgs/Header    header


float64 flow1
float64 flow2
float64 flow3

int16 osc

int16 valve1
int16 valve2
int16 valve3


